docs/extending.md

```diff
@@ -0,0 +1,19 @@
+# Extending the adapter/serializer
```

Comment:
Good point.  I removed everything usage-related from the README—we can keep all that in the docs going forward.

I did not copy the instructions to change how URLs are transformed.  This is already documented in the Ember docs, pretty easy to figure out with the blueprints, and I can't imagine it being something people will actually do too often.  If it becomes a pain point, we can add it back somewhere.

I also didn't copy the warning about the coalesceFindRequests.  It is disabled by default in ED anyway, and I plan on tackling it very soon.